index.js

```diff
@@ -223,19 +224,23 @@ async function main() {
       core.debug(inspect(data));
     }
   
-    if (inputs.reviewers) {
+    if (inputs.reviewers || inputs.team_reviewers) {
       core.debug(`Adding reviewers: ${inputs.reviewers}`);
+      core.debug(`Adding team reviewers: ${inputs.team_reviewers}`);
       const reviewers = inputs.reviewers.trim().split(/\s*,\s*/);
+      const team_reviewers = inputs.team_reviewers.trim().split(/\s*,\s*/);
```

Comment:
are you sure the inputs are strings when they are not set?

Reply:
Updated to build up the parameters with null checks: https://github.com/gr2m/create-or-update-pull-request-action/pull/493/commits/57e75b2afefdb90358d8faa0a181626e707722ca